clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/handlers/BasicGoogleDeployHandler.groovy

```diff
@@ -204,16 +204,46 @@ class BasicGoogleDeployHandler implements DeployHandler<BasicGoogleDeployDescrip
       : null
 
     def migCreateOperation
+    def instanceGroupManager = new InstanceGroupManager()
+        .setName(serverGroupName)
+        .setBaseInstanceName(serverGroupName)
+        .setInstanceTemplate(instanceTemplateUrl)
+        .setTargetSize(description.targetSize)
+        .setTargetPools(targetPools)
+        .setAutoHealingPolicies(autoHealingPolicy)
+
+    def hasBackendServices = description.instanceMetadata &&
+        description.instanceMetadata.containsKey(GoogleServerGroup.View.BACKEND_SERVICE_NAMES)
+
+    if (hasBackendServices && (description?.loadBalancingPolicy || description.source.serverGroupName))  {
+      NamedPort namedPort = null
+      def sourceManagerName = description.source.serverGroupName
+      if (sourceManagerName) {
+        def sourceInstanceGroupManager
+        if (isRegional) {
+          sourceInstanceGroupManager = compute.regionInstanceGroupManagers().get(project, description.source.region, sourceManagerName).execute()
+        } else {
+          // TODO(jacobkiefer): This will work only if the new zone is the same as the old zone. This will only be a problem for ad-hoc.
+          sourceInstanceGroupManager = compute.instanceGroupManagers().get(project, description.zone, sourceManagerName).execute()
+        }
+        if (!sourceInstanceGroupManager) {
+          GCEUtil.updateStatusAndThrowNotFoundException("Could not locate source group manager ${sourceManagerName} to update named port.", task, BASE_PHASE)
+        }
+        namedPort = sourceInstanceGroupManager.namedPorts.find { it.name == GoogleHttpLoadBalancingPolicy.HTTP_PORT_NAME }
+      } else {
```

Comment:
Yep, I'll fix this so we can mix the specifications: i.e. clone but specify load balancing policy.